### DISCUSSION TITLE
How to increase image number >32?

### DISCUSSION THREAD

**Discussion by consistentbenny** [lllyasviel/Fooocus#1537] [Q&A] [answered]
Hi there!

Thanks for this awesome piece of software, first of all!

I do have a question, as for my use case I often have to run the same prompt multiple times to receive the amount of images I need to make a proper selection, which is usually >100 images.

I use Fooocus on a cloud GPU server, which I spin up every single time from scratch. Is there any line of code that I can enter before opening the Gradio link that allows me to create more than 32 images, which is the maximum I can define using the GUI?

This would make things a lot easier, but I haven't figured out a way to do so yet...

Thanks for any advice! 🙏

**Comment by siriume** [comment]
Modify this parameter(maximum, 32) to your desired value
https://github.com/lllyasviel/Fooocus/blob/b0df0d57f62636fd8670d8f64482b3cde2aca05c/webui.py#L229

**Reply by consistentbenny** [reply]
Thank you! 🙏

**Reply by jackichad** [reply]
Perhaps things have changed a bit from when this reply was posted about a month ago, but when I look in my webui.py I have the following at line 232 instead of 229:

`image_number = gr.Slider(label='Image Number', minimum=1, maximum=modules.config.default_max_image_number, step=1, value=modules.config.default_image_number)`

Is this still something simple that can be adjusted?

Thanks!

**Reply by consistentbenny** [reply, marked as answer]
Yes, this has changed!

Instead of changing the file `webui.py`, this can now also be adjusted with the `config.txt` file. Simply add the following line to it and adjust the number to your needs:

```
{
    "path_checkpoints": "C:\\Path\\Fooocus\\models\\checkpoints",
    "path_loras": "C:\\Path\\Fooocus\\models\\loras",
    "path_embeddings": "C:\\Path\\Fooocus\\models\\embeddings",
    "path_vae_approx": "C:\\Path\\Fooocus\\models\\vae_approx",
    "path_upscale_models": "C:\\Path\\Fooocus\\models\\upscale_models",
    "path_inpaint": "C:\\Path\\Fooocus\\models\\inpaint",
    "path_controlnet": "C:\\Path\\Fooocus\\models\\controlnet",
    "path_clip_vision": "C:\\Path\\Fooocus\\models\\clip_vision",
    "path_fooocus_expansion": "C:\\Path\\Fooocus\\models\\prompt_expansion\\fooocus_expansion",
    "path_outputs": "C:\\Path\\Fooocus\\outputs",
    "default_max_image_number": 100
}
```

If you want to add other parameters to that config file, make sure to **add a comma at the end of each line, except for the last one!**

**Reply by jackichad** [reply]
That's perfect! Thank you!

**Comment by juliux32** [comment]
Just out of curiosity... what you are generating that you need 100+ images just to get a proper selection? Usually, I get a good image from less than 10 images 😅 or you are just that picky?

**Reply by consistentbenny** [reply]
I'm not necessarily picky, but I need a certain quantity of good images and for some creative use cases that involve certain parts that the image generation really struggles with (mouth, teeth, hands, fingers, etc.) often >90% of images are not usable. It is easier then to just create a ton upfront and sort through after.

**Reply by Pal1ndrome** [reply]
I realize I'm necroing an old thread...but I also like to increase the number to use wildcards to generate a large batch of "random" images based on the parameters in my wildcards, such as background, styles, and subjects.

**Reply by lgwjames** [reply]
> I realize I'm necroing an old thread...but I also like to increase the number to use wildcards to generate a large batch of "random" images based on the parameters in my wildcards, such as background, styles, and subjects.

there is a post aboult bulk images

**Comment by Trevor12-dev** [comment]
how can i reload the gpu for the foocus
